### ISSUE TITLE
Accessibility - Main Screen - Poor Color Contrast for the Right-pointing Arrows

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Child-Labor#123]
On the main screen for the application, I noticed that the light gray right-pointing arrows ( #C4C4C4 ) on the white background above the image only has a color contrast ratio of 1.7:1. So I'm going to add a new issue to that, so we have it for our next build to work on. Also, the divider lines between the three items have the same issue.

I'm assuming we have the same issue for Android and web.

Please see the attached screenshot.

![Child Labor - iOS - main screen - poor color contrast of the right-pointing arrows](https://user-images.githubusercontent.com/10423487/73091771-233fcb80-3ea9-11ea-8d4a-56e245f477c1.jpeg)

**Comment by jfc3-dol** [comment]
@Abychalil,

On the main screen for the application, I still notice that the light gray right-pointing arrows ( #C4C4C4 ) on the white background above the image only has a color contrast ratio of 1.7:1. Also, the divider lines between the three items have the same issue.

I'm assuming we have the same issue for Android and web.

Please see the attached screenshot.

![ialb - sweat adn toil - home page - color contrast issues](https://user-images.githubusercontent.com/10423487/88321164-b9d03680-ccec-11ea-859c-585b2a8e5678.png)


**NOTE** - That now the Application name of "Sweat & Toil" text is black and does not have enough color contrast against the blue background so that I will create a new issue for that problem.

**Comment by jfc3-dol** [comment]
ILAB developer/team response

This is an inbuilt function of Xcode, and Apple doesn’t allow developers to override its default color. If you check your iPhone -> Settings ( attached "iphone setting screen showing light colored right pointing arrows.jpg" ), you’ll find the same right arrow color as our app (attached file App-Screenshot.png). We have a solution is to replace the default “Disclosure Indicator” to a right arrow (attached chevron.png). App-SS-Chevron.png file displays the right arrow in the app. If we are going with this approach, we’ll need the right arrow image from the ILAB design team. This Chevron is a screenshot of MS word shapes.

jfc iii - response

It's hard to believe Xcode/Apple won't let people change the color contrast of an item, or is it because it's from a template from them? 

If you use a darker image, you will have to be careful with ALT text and how they might be read by VoiceOver.

Maybe we can have a call to talk about it because they probably shouldn't have the same ALT text for each one.

Please see the attached screenshot ILAB mentioned above.

![App-SS-Chevron](https://user-images.githubusercontent.com/10423487/88696116-d5628500-d0d0-11ea-8838-55a4c5f23e68.png)


![chevron](https://user-images.githubusercontent.com/10423487/88696117-d5fb1b80-d0d0-11ea-87e4-2f9fa66deed8.png)


![App-screenshot](https://user-images.githubusercontent.com/10423487/88696119-d5fb1b80-d0d0-11ea-97d2-3515fe59be8d.png)


![iphone setting screen showing light colored right pointing arrows](https://user-images.githubusercontent.com/10423487/88696121-d693b200-d0d0-11ea-8082-454ea65cfc50.jpg)

**Comment by jfc3-dol** [comment]
@Abychalil,

Now that the right-pointing arrows on the home page along with the main Countries and Goods screens no have enough color contrast to pass, I'm going to close this issue.
